pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -2347,6 +2347,41 @@ public class Baz {        // poor, missing an equals() method
     }
 }
 
+public class Foo {        // perfect, both methods provided
+    public boolean equals(Object other) {
+      // do some comparison
+    }
+    public int hashCode() {
+      // return some hash value
+    }
+}
+]]>
+        </example>
+    </rule>
+    <rule name="MissingEqualsOnComparable"
+          language="java"
+          since="0.4"
+          message="Ensure you override both equals() and hashCode()"
+          class="net.sourceforge.pmd.lang.java.rule.errorprone.MissingEqualsOnComparableRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_errorprone.html#missingequalsoncomparable">
+        <description>
+Override both public boolean Object.equals(Object other), and public int Object.hashCode(), or override neither.  Even if you are inheriting a hashCode() from a parent class, consider implementing hashCode and explicitly delegating to your superclass.
```

Comment:
This looks like copy/paste from the other rule - we should explain here, why one should implement equals if Comparable is implemented (consistency between equals and compareTo, potential problems if instances are collected in a sorted collection like SortedSet, ...)

Reply:
yes, thx. done.